L/LibAMVW/build_tarballs.jl

```diff
@@ -0,0 +1,98 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "LibAMVW"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://people.cs.kuleuven.be/~raf.vandebril/homepage/software/AMVW.tar", "37000a9a5a83677fc18203bc6ae81657d74103b11eeb2257d3e68802731a85ae")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+if [[ ${target} == *"mingw"* ]]; then
+    SLIB=dll
+    LIBDIR=bin
+elif [[ ${target} == *"darwin"* ]]; then
+    SLIB=dylib
+    LIBDIR=lib
+else
+    SLIB=so
+    LIBDIR=lib
+fi
```

Comment:
We already have all these variables: https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Automatic-environment-variables-1
```suggestion
```